app/src/main/java/com/infomaniak/mail/ui/main/folder/ThreadListViewModel.kt

```diff
@@ -65,18 +66,16 @@ class ThreadListViewModel @Inject constructor(
     }
 
     fun checkWebViewVersion(canShowWebViewOutdated: Boolean) {
-        isWebViewOutdated.value = if (canShowWebViewOutdated) {
+        val (webViewVersionName, webViewMajorVersion) =
             WebViewCompat.getCurrentWebViewPackage(appContext)?.versionName?.let { versionName ->
-                val majorVersion = runCatching {
-                    versionName.substring(0, versionName.indexOfFirst { it == '.' }).toInt()
-                }.getOrElse {
-                    0
-                }
-                majorVersion < WEBVIEW_MIN_VERSION
-            } ?: true
-        } else {
-            false
-        }
+            val majorVersion = runCatching { versionName.substringBefore('.').toInt() }.getOrDefault(0)
+            versionName to majorVersion
+        } ?: (null to 0)
+
+        val needToUpdateWebView = canShowWebViewOutdated && webViewMajorVersion < WEBVIEW_MIN_VERSION
+
+        SentryDebug.sendWebViewVersionName(webViewVersionName, webViewMajorVersion)
```

Comment:
This Sentry will trigger at every ThreadList `onStart()`.
Is it what we want ?

Reply:
First, check the WebView version (always do it).
Then :
- If the version is wrong, send Sentry.
- If the version is wrong + the user never clicked "Later", display the banner.